test/e2e/framework/kubeletconfig.go

```diff
@@ -0,0 +1,131 @@
+/*
```

Comment:
Could you implement this as `test/e2e/framework/kubelet/config.go`?

If implementing this as `test/e2e/framework/kubeletconfig.go`, this file is defined as core part of e2e test framework.
This file imports `e2ekubectl` which is defined as subpackage of e2e test framework, and that means invalid dependency as https://github.com/kubernetes/kubernetes/issues/81245
By implementing as ` test/e2e/framework/kubelet/config.go` we can avoid such thing.

Reply:
/cc @oomichi

Reply:
Sure thing @oomichi , on it